config-bootstrapper/src/main/resources/configs/config-bootstrapper/entity-service/hypertrace/2_entitytypesv2.conf

```diff
@@ -0,0 +1,43 @@
+version = 1
+name = entitytypesv2
+commands = [
+	{
+    upgrade : [
```

Comment:
The entity types aren't actually ready to migrate over yet - we'll want to do these one by one.

Reply:
Okay.. i'll leave it to you to add later then.